include/tusb_config.h

```diff
@@ -0,0 +1,113 @@
+/**
```

Comment:
What is different from the tinyusb library? I really don't want to have a copy of tinyusb in this repo. We don't get any of the goodness of shared code if we have a completely separate copy of it. 

I can't tell how much was changed, but if you had to re-write a significant amount of it, let's discuss. it'd be better to fork the TinyUSB git repo so that we can merge fixes.

Reply:
Actually TinyUSB is included in the plaftform I am using, but I couldn't get the tinyusb header file to include. What I can do is in the plaformio.ini section for Quakka, reference the specific commit on github for TinyUSB that way the TinyUSB code will no longer be included in the lib directory.  I will move the circular queue I wrote to the Rabbit Hole repo and link to it the same way.

Reply:
If the circular queue is truly different from the original one I used, I have no objections with it being in this repo. (TBH, I didn't look that closely.)

Totally up to you though. If you want to re-use it for other projects, it can go in a different repo.

Just curious - why did you switch to a different one?